R/dataPrepRast.R

```diff
@@ -0,0 +1,131 @@
+#' Crop, reproject, and mask a raster map
+#'
+#' This script reprojects and crop a raster using a raster or polygon object.
+#'
+#' @param x   A \code{RasterLayer} object or the name of one.
```

Comment:
If `x` can only be a `RasterLayer`, this needs to be enforced early on in the function (*e.g.*, using `stopifnot()`), although using more formal methods for this function would be more useful, as we can extend this to work for other `Raster*` (or even `Spatial*`) objects.